src/models/Erc721ContractModel.ts

```diff
@@ -0,0 +1,48 @@
+const mongoose = require("mongoose");
+const mongoosePaginate = require("mongoose-paginate");
+const Schema = mongoose.Schema;
+
+const ERC721ContractScheme = new Schema({
+    address :  {
+        type: String,
+        index: true
+    },
+    name: {
+        type: String,
+        required: true,
+        index: true
+    },
+    symbol: {
+        type: String,
+        required: true,
+        index: true
+    },
+    totalSupply: {
+        type: String,
+        required: true
+    },
+    implementsERC721: {
+        type: Number,
+        required: true
+    },
+    verified: {
+        type: Boolean,
+        default: false
+    },
+    enabled: {
+        type: Boolean,
+        default: true,
+        index: true
+    }
+}, {
+    versionKey: false,
+});
+
+// indices
+ERC721ContractScheme.index({address: 1}, {name: "contractAddressIndex"});
```

Comment:
no need to specify the name, it will do it automatically.

Reply:
Don't know how to change it as it is copied from ERC20ContractScheme

Reply:
Removed from here. Actually we don't compound indexes in ERC20ContractScheme as well (http://mongoosejs.com/docs/guide.html#indexes), will remove it later.